docs/content/en/docs-dev/user-guide/managing-piped/configuration-reference.md

```diff
@@ -179,6 +179,8 @@ Must be one of the following structs:
 | address | string | The address of Datadog API server. Only "datadoghq.com", "us3.datadoghq.com", "datadoghq.eu", "ddog-gov.com" are available. Defaults to "datadoghq.com" | No |
 | apiKeyFile | string | The path to the api key file. | Yes |
 | applicationKeyFile | string | The path to the application key file. | Yes |
+| apiKey | string | Optional: API Key for Datadog API server. | No |
+| applicationKey | string | Optional: Application Key for Datadog API server | No |
```

Comment:
```suggestion
| apiKeyData | string | Optional: API Key for Datadog API server. | No |
| applicationKeyData | string | Optional: Application Key for Datadog API server | No |
```
should add suffix `Data` to unify with other options like [this](https://github.com/pipe-cd/pipecd/blob/master/docs/content/en/docs-dev/user-guide/managing-piped/configuration-reference.md?plain=1#L218)

Reply:
Also as the piped convention, the field which holds key/token data should hold the base64 encode version of the key/token, so I think we should do the same here as well.

ref: https://pipecd.dev/docs-v0.44.x/user-guide/managing-piped/configuration-reference/#:~:text=Yes-,pipedKeyData,-string

Reply:
yes, better if we hold key/token as base64 when direct on yaml file. Implemented that! 🙏